apps/studio/src/server/modules/folder/folder.router.ts

```diff
@@ -315,4 +320,75 @@ export const folderRouter = router({
 
       return { title, ...indexPage }
     }),
+
+  insertChildpageBlock: protectedProcedure
```

Comment:
question: hmm any reason why this new mutation was created? Seems like it is quite a duplicate with the `updatePageBlob` procedure in page.router, and frontend should be able to be adjusted slightly to use that procedure instead.

Reply:
also can, added because i wanted this to be backend focused (ie, we always add the same block) but i'll swap to `updatePageBlob` instead!